src/main/java/org/broadinstitute/hellbender/utils/Utils.java

```diff
@@ -1137,4 +1141,128 @@ public static void forceJVMLocaleToUSEnglish() {
         final List<T> sorted = values.stream().sorted().collect(Collectors.toList());
         return sorted.get(sorted.size() / 2);
     }
+
+    public static class LineIterator implements AutoCloseable, Iterable<String> {
```

Comment:
Can you move this class into its own file in the `utils.nio` package?

Reply:
sure